schema/example.go

```diff
@@ -0,0 +1,173 @@
+package schema
+
+func NewExample() Schema {
+	location := locationSchema{
+		Lat:  35.712758,
+		Long: 51.392114,
+	}
+
+	score := scoreSchema{
+		Type:  "GPA",
+		Value: "3.4",
+	}
+
+	resource := resourceSchema{
+		URL:   "http://www.example.com/my-example-slides/",
+		Label: "Slides",
+	}
+
+	core := coreSchema{
+		Name:              "John Doe",
+		Title:             "Software Engineer",
+		Image:             "example.com/Abcxyz",
```

Comment:
General note: We should probably use a base64 image instead so it's stored within the schema and not externally pulled in. Within the front end or the planned helper library we would then make it possible to "upload" or add a link to an image and the helper would pull it down and attach it base64 encoded to the schema.